writerService_test.go

```diff
@@ -1,26 +1,26 @@
 package main
 
 import (
-	"bytes"
-	"fmt"
-	"github.com/stretchr/testify/assert"
+	"github.com/go-playground/assert/v2"
+	assert2 "github.com/stretchr/testify/assert"
+	"os"
 	"testing"
 )
 
-func TestMeasurementsWriterService_Start_OutputsCorrectStringAndCloses(t *testing.T) {
-	ws := MeasurementsWriterService{}
-	publish := make(chan Measurement)
-	measurement := Measurement{
-		Id:    1,
-		Value: 2,
+func TestNewMeasurementsWriterService_ReturnsNewWriterServiceAddress(t *testing.T) {
+	actual := NewMeasurementsWriterService(os.Getenv("INFLUX_ADDRESS"))
```

Comment:
This is not a good idea. Test shouldn't depend on external database if you don't provide it with the test, e.g. with docker swarm. If I were you, I would remove this test. It doesn't really test much